server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationFileCreator.java

```diff
@@ -0,0 +1,173 @@
+package org.opentosca.toscana.plugins.cloudformation;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.opentosca.toscana.core.plugin.PluginFileAccess;
+import org.opentosca.toscana.plugins.scripts.BashScript;
+import org.opentosca.toscana.plugins.scripts.EnvironmentCheck;
+
+import org.apache.commons.io.Charsets;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+
+/**
+ * Class for building scripts and copying files needed for deployment of cloudformation templates.
+ */
+public class CloudFormationFileCreator {
+    static final String CLI_COMMAND_CREATESTACK = "aws cloudformation deploy ";
```

Comment:
maybe you want to add a modifier.

Reply:
This was done to allow access for the test classes (no modifier makes them package-private).
I'm not sure if this is a good or bad practice though, so I don't know if it's better the add separate Strings to the test classes or to use the same for both like I did it here.